package.json

```diff
@@ -1,92 +1,106 @@
-        "apify",
-        "api",
-        "apifier",
-        "crawler",
-        "scraper"
-    ],
-    "author": {
-        "name": "Apify",
-        "email": "support@apify.com",
-        "url": "https://apify.com"
-    },
-    "contributors": [
-        "Jan Curn <jan@apify.com>",
-        "Marek Trunkát <marek@apify.com>",
-        "Ondra Urban <ondra@apify.com>",
-        "Jakub Drobník <jakub.drobnik@apify.com>"
-    ],
-    "license": "Apache-2.0",
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/apifytech/apify-client-js"
-    },
-    "bugs": {
-        "url": "https://github.com/apifytech/apify-client-js/issues"
-    },
-    "homepage": "https://apify.com/docs/sdk/apify-client-js/latest",
-    "files": [
-        "src",
-        "dist"
-    ],
-    "browser": {
-        "agentkeepalive": false,
-        "fs": false,
-        "os": false,
-        "stream": false,
-        "util": false,
-        "zlib": false
-    },
-    "scripts": {
-        "build": "npm run clean && npm run build-browser",
-        "build-docs": "npm run build-toc && node docs/build_docs.js",
-        "build-toc": "markdown-toc docs/README.hbs -i",
-        "test": "npm run build && jest",
-        "prepare": "npm run build",
-        "prepublishOnly": "(test $CI || (echo \"Publishing is reserved to CI!\"; exit 1))",
-        "clean": "rm -rf dist",
-        "lint": "eslint src test",
-        "lint:fix": "eslint --fix src test",
-        "build-browser": "webpack"
-    },
-    "dependencies": {
-        "@apify/consts": "^1.0.0",
-        "@apify/log": "^1.0.0",
-        "agentkeepalive": "^4.1.3",
-        "async-retry": "^1.3.1",
-        "axios": "^0.21.1",
-        "content-type": "^1.0.4",
-        "ow": "^0.23.0"
-    },
-    "devDependencies": {
-        "@apify/eslint-config": "^0.1.2",
-        "@babel/cli": "^7.11.6",
-        "@babel/core": "^7.11.6",
-        "@babel/preset-env": "^7.11.5",
-        "@babel/register": "^7.11.5",
-        "@types/express": "^4.17.8",
-        "@types/fs-extra": "^9.0.4",
-        "@types/jest": "^26.0.14",
-        "@types/node": "^14.11.5",
-        "apify": "^1.0.0",
-        "babel-loader": "^8.1.0",
-        "body-parser": "^1.19.0",
-        "brotli-webpack-plugin": "^1.1.0",
-        "compression": "^1.7.4",
-        "eslint": "^7.10.0",
-        "express": "^4.17.1",
-        "fs-extra": "^9.0.1",
-        "jest": "^26.5.0",
-        "jsdoc-to-markdown": "^6.0.1",
-        "markdown-toc": "^1.2.0",
-        "morgan": "^1.10.0",
-        "puppeteer": "^5.5.0",
-        "webpack": "^4.44.2",
-        "webpack-cli": "^3.3.12"
-    }
+	"name": "apify-client",
+	"version": "1.3.0",
+	"description": "Apify API client for JavaScript",
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
+	"types": "dist/index.d.ts",
+	"browser": "dist/bundle.js",
+	"unpkg": "dist/bundle.js",
+	"exports": {
+		".": {
+			"import": "./dist/index.mjs",
+			"require": "./dist/index.js"
+		}
+	},
```

Comment:
I think this was causing issues for us in `apify-shared`. The Apify Web could not be built because webpack was not finding the right files. I think we fixed it, but another user on Discord complained that he can't use the client with a `create-react-app`, where he does not have access to Webpack config...

Reply:
I've added entries to the package.json that should point browser compilers to use our bundle... Now, I cannot test it sadly, but it shouldn't cause issues..